EIPS/eip-1.md

```diff
@@ -64,7 +64,7 @@ EIPs can also be superseded by a different EIP, rendering the original obsolete.
 
 The possible paths of the status of EIPs are as follows:
 
-![EIP Process](eip-1/process.png)
+![EIP Process](/assets/eip-1/process.png)
```

Comment:
Please don't use absolute paths - this will break for anyone running it on theirname.github.io.

Reply:
ok I'll fix it to `../assets/eip-1/process.png` (and the other links too)